src/application.rs

```diff
@@ -257,7 +278,7 @@ impl SwayOSDApplication {
 			}
 		}
 
-		return self.app.run();
+		self.app.run().into()
```

Comment:
What's the difference? I'm a bit of a Rust noob 😅

Reply:
there are 2 changes here.
First in rust you don't need to type `return [var];` if you just write `[var]` without a semicolon it returns the value. return should only be used inside loops, if statements etc. to skip the rest of the code execution.

The into() is there because of an update of the GTK library. Basicially you are returning I think it was some enum but in the newer version of the library they want an integer and this just converts the enum to an integer

Reply:
> The into() is there because of an update of the GTK library. Basicially you are returning I think it was some enum but in the newer version of the library they want an integer and this just converts the enum to an integer

Sorry, this is what I was asking for. Knew about the lack of return feature. Thanks :)